src/components/exchange/index.js

```diff
@@ -0,0 +1,31 @@
+import { connect } from 'react-redux';
+
+import Exchange from './Exchange';
+
+const getStockExchange = state => {
```

Comment:
This has a very similar format to 
```javascript
const getSelectedStockDataFromState = state => {
  const symbol = state.selectedStock;
  const selectedStock = state.stocks[symbol];
  return selectedStock.stockData;
};
```
in utils/formatting/dataItem.js.

Makes me think we should define
```javascript
const getSelectedStockValueForKey = (state, key) => {
  const symbol = state.selectedStock;
  const selectedStock = state.stocks[symbol];
  return selectedStock[key];
};
```
in utils somewhere and use that to get rid of duplication.

Also just a note, **getStockExchange** and **getSelectedStockDataFromState** are not consistently named.